ciliumconfig.conformance.yaml

```diff
@@ -36,3 +35,4 @@ spec:
     serviceMonitor: {enabled: false}
   hubble:
     tls: {enabled: false}
+  sessionAffinity: true
```

Comment:
just calling out i had to add this setting for v1.13. ref: https://github.com/cilium/cilium/commit/ec41c3d05d8a90c6139688f37b261f779fbe2ddf